__tests__/app.test.js

```diff
@@ -38,5 +36,72 @@ describe("/api/topics", () => {
             });
         });
     });
-    
+    describe('/api/articles/:article_id', () => {
+        test("200: response with a status of 200", () => {
+            return request(app).get("/api/articles").expect(200)
+        });
+        test('200: should return a specific article by ID', () => {
+            return request(app).get('/api/articles/1')
+            .then(({ body }) => {
+                const article = body.article
+                expect(article).toMatchObject({
+                    author: expect.any(String),
```

Comment:
If we request a specific article, we know a little more than just the id. We should be expecting a very specfic string for the title, body, etc.